### ISSUE TITLE
RockyLinux 8.9  {"msg": "You need to install \"jmespath\" prior to running json_query filter"}

### ISSUE THREAD

**Issue by xjdata** [Vonng/pigsty#428]
#389  与这个类似   但我的环境  RockyLinux 8.9 装了所有 jmespath  还是在报错。。。

```bash
[zz@zz-pg-1 ~]$ sudo dnf install -y  python3-jmespath python38-jmespath python39-jmespath python3.11-jmespath python3.12-jmespath
上次元数据过期检查：0:23:02 前，执行于 2024年07月02日 星期二 18时14分28秒。
软件包 python3-jmespath-0.9.0-11.el8.noarch 已安装。
软件包 python38-jmespath-0.10.0-2.el8.noarch 已安装。
软件包 python39-jmespath-1.0.0-7.el8.noarch 已安装。
软件包 python3.11-jmespath-1.0.1-1.el8.noarch 已安装。
软件包 python3.12-jmespath-1.0.1-1.el8.noarch 已安装。
依赖关系解决。
无需任何处理。
完毕！
```

```bash
ok: [192.168.1.83] => {
    "msg": "nodename: zz-redis-sentinel-3, node_cluster: nodes, spec: c2m2, os: rocky.8.x86_64 el8:rpm,    [REDIS: ez-redis-sentinel]    "
}

TASK [pg_id : get pgsql identity] **********************************************************************************************************************************************************
skipping: [192.168.1.50]
fatal: [192.168.1.63]: FAILED! => {"msg": "You need to install \"jmespath\" prior to running json_query filter"}
fatal: [192.168.1.61]: FAILED! => {"msg": "You need to install \"jmespath\" prior to running json_query filter"}
skipping: [192.168.1.81]
skipping: [192.168.1.82]
fatal: [192.168.1.62]: FAILED! => {"msg": "You need to install \"jmespath\" prior to running json_query filter"}
skipping: [192.168.1.83]

```

**Comment by Vonng** [comment]
Seems that the default jmespath used by ansible is switching from

`python3.11-jmespath` to `python3.12-jmespath`

**Comment by Vonng** [comment]
will be fixed in Pigsty v2.8
